cmd/pebble/main.go

```diff
@@ -29,6 +29,17 @@ type config struct {
 	}
 }
 
+func setConfigDefaults() (config) {
+	var conf config
+	conf.Pebble.ListenAddress = "0.0.0.0:14000"
```

Comment:
Add a comment showing readers that these values are identical to the values in `/test/config/pebble-config.json`, and that they should be kept in sync.